src/videotoolbox.cs

```diff
@@ -278,6 +304,22 @@ interface VTCompressionProperties {
 		[Mac (10,13), iOS (11,0), TV (11,0)]
 		[Export ("EncoderId")]
 		string EncoderId { get; set; }
+		
+		[Mac (10,15), iOS (13,0), TV (13,0)]
+		[Export ("TargetQualityForAlpha")]
+		float TargetQualityForAlpha { get; set; }
+		
+		[Mac (10,9), iOS (13,0), TV (13,0)]
+		[Export ("GammaLevel")]
+		float GammaLevel { get; set; }
+		
+		[Mac (10,15), iOS (13,0), TV (13,0)]
+		[Export ("AlphaChannelMode")]
+		string AlphaChannelMode { get; set; }
+		
+		[Mac (10,15), iOS (13,0), TV (13,0)]
+		[Export ("UsingGpuRegistryId")]
+		uint UsingGpuRegistryId { get; }
```

Comment:
It was only ```read``` in apple lib, does it means I only need ```get;```?

Reply:
Correct, that's a read-only property.